grails-app/views/consumption/_filters.gsp

```diff
@@ -20,64 +20,83 @@
         <g:hiddenField name="parametersHash" value="${command?.generateParametersHash()}"/>
 
         <div class="box">
-            <h2><warehouse:message code="consumption.parameters.label" default="Parameters"/></h2>
+            <h2><g:message code="consumption.parameters.label" default="Parameters"/></h2>
```

Comment:
Why are we changing theses? I thought we agreed not to change the namespace.

Reply:
I remember at some point you wanted to get rid of `warehouse:message` so having an occasion I decided to change it to `g:message`. If you would like to stick with `warehouse:message` for now, I can bring it back

Reply:
No, I don't want to change anything with these taglibs in the develop branch because it adds more work for the rebase on the Grails 3 branch.

Reply:
So revert all of those. If there's an issue with warehouse:message we can discuss that but please refrain from making small refactorings like this. It can add a lot of complexity to the rebase so I want to limit the changes we make to existing code.

Reply:
ok, sure